apps/backend/backend-main/src/modules/auth/auth.service.ts

```diff
@@ -54,7 +54,7 @@ export class AuthService {
     logger.debug(
       `AuthService.signupInitiate: Initiating signup for ${phoneNumber}`,
     );
-    const existingUser = await dbRepo(User).findOne({ where: { phoneNumber } });
+    const existingUser = await dbRepo(User).findOneOrFail({ where: { phoneNumber } });
```

Comment:
```suggestion
    const existingUser = await dbRepo(User).findOne({ where: { phoneNumber } });
```

Pointless update because we want to return the onboarding token if the user doesn't exist.